### PR TITLE
72/endpoint join users to goals

### PR DESCRIPTION
**Pull request by nicksantoro**

### CODE REVIEW

File: src/controllers/goals.js [hunk 1]
```diff
@@ -58,12 +58,27 @@ const likeGoal = (req, res, next) => {
 
 }
 
+const joinGoal = (req, res, next) => {
+  let goals_id = req.params.id;
+  console.log(goals_id)
```
Comment (developerX) [inline src/controllers/goals.js:63]: remove me

File: src/controllers/goals.js [hunk 2]
```diff
@@ -58,12 +58,27 @@ const likeGoal = (req, res, next) => {
...
+  let { users_id } = req.body;
+  console.log("users", users_id)
```
Comment (developerX) [inline src/controllers/goals.js:65]: remove me

File: src/controllers/goals.js [hunk 3]
```diff
@@ -58,12 +58,27 @@ const likeGoal = (req, res, next) => {
...
+  console.log("goals", goals_id)
```
Comment (developerX) [inline src/controllers/goals.js:66]: remove me

File: src/controllers/users.js [hunk 4]
```diff
@@ -60,11 +60,11 @@ const updateUserById = (req, res, next) => {
 const loginUser = async (req, res, next) => {
   let payload = req.body;
   if (!payload.password || !payload.email) return res.status(400).json({ error: "username and password required" })
-  const userExists = await model.getUserByEmail(payload.email)
-  if (userExists.error) return res.status(404).json({ error: "user does not exist" })
-  const isValid = await bcrypt.compare(payload.password, userExists.password);
+  const user = await model.getUserByEmail(payload.email)
```
Comment (developerX) [inline src/controllers/users.js:63]: wasn't this change in another PR? Why is it in here? Is this what you were referencing to on your slack message?

**Review by developerX — APPROVED** [review]
A few small changes and questions, once updated please ship it!

File: src/controllers/users.js (hunk 4 above)
Comment (carlcorsini) [reply src/controllers/users.js:63]: I remember, it is because Nick changed the variable `userExists` to `user`

**Pull request merged**